tests/examples/opinions/united_states/afcca_example.compare.json

```diff
@@ -1,332 +1,32 @@
 [
   {
-    "case_dates": "2017-02-17",
-    "case_names": "United States v. Roblero",
-    "download_urls": "tests/examples/opinions/united_states/afcca_opinions/cp/roblero-38874.u.pdf",
+    "case_dates": "2023-01-11",
+    "case_names": "United States v. Goldsmith",
+    "download_urls": "tests/examples/opinions/united_states/afcca_opinions/cp/goldsmith_-_40148_u_1066513.pdf",
     "precedential_statuses": "Unpublished",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "ACM 38874",
-    "case_name_shorts": "Roblero"
+    "docket_numbers": "40148",
```

Comment:
Hm, big diff in docket number format here, no?

Reply:
yes, I saw that, but the docket numbers look like they include ACM to me.

Reply:
but, on the website, I think its less clear. 
<img width="255" alt="Screenshot 2023-01-19 at 10 41 16 AM" src="https://user-images.githubusercontent.com/6464529/213486665-606770e4-d153-4035-bb22-69e52622cba6.png">

Reply:
Both seem acceptable. So be it.